Sources/MapboxMaps/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandler.swift

```diff
@@ -1,11 +1,15 @@
 import UIKit
 
+internal protocol DoubleTapToZoomInGestureHandlerProtocol: GestureHandler {
+    var focalPoint: CGPoint? { get set }
```

Comment:
Wherefrom this term come? It sounds wrongly placed a bit for me. 
Does `anchorPoint` work?